homepage/models.py

```diff
@@ -54,13 +54,23 @@ def __str__(self):
         return self.user.username
 
 
+class Professor(models.Model):
+    name = models.CharField(max_length=100)
+
+    def __str__(self):
+        return self.name
+
+
 class Review(models.Model):
     course = models.ForeignKey(Course, on_delete=models.CASCADE)
     user = models.ForeignKey(AppUser, on_delete=models.CASCADE)
     rate = models.SmallIntegerField()
     date = models.DateTimeField(default=timezone.now)
     content = models.TextField()
     course_load = models.SmallIntegerField()
+    likes_num = models.SmallIntegerField(default=0)
+    professor_id = models.ForeignKey(Professor, on_delete=models.CASCADE, null=True, blank=True)

```

Comment:
can I recommend changing the name to `professor` instead of `professor_id`, 
or adding this to the description: ` db_column='professor_id'`
since django automatically adds '_id' to the end of every FK column.

Reply:
Name changed to ```professor```